tools/ci.sh

```diff
@@ -692,6 +697,27 @@ function ci_unix_qemu_arm_run_tests {
     (cd tests && MICROPY_MICROPYTHON=../ports/unix/build-coverage/micropython ./run-tests.py --exclude 'vfs_posix.*\.py')
 }
 
+function ci_unix_qemu_riscv64_setup {
+    for repository in jammy jammy-updates jammy-security
```

Comment:
To make this future-proof, we could `. /etc/os-release` at the beginning of this file. Then we can use `$VERSION_CODENAME` instead of `jammy`.